docs/News.rst

```diff
@@ -0,0 +1,59 @@
+.. _News:
+
+###############
+Creating a News
+###############
+
+From a space's acivity stream, you are able to share a news with the space’s members. 
```

Comment:
What do you think about this sentence: 
The feature news allows you to easily write, broadcast, pin and share communication content.

Reply:
The term "feature" is weird here IMO. It could be something like "Publishing a News allow to publish communication content.". Maybe we should develop a bit more, but for the moment we cannot do much more since we cannot pin, we cannot share, ..., it will be done in next stories.

Reply:
I agree, we'll have to develop more the news description in next storiet